draft-ietf-httpbis-digest-headers.md

```diff
@@ -501,6 +500,47 @@ to collision attacks [CMU-836068].
 The SHA algorithm is NOT RECOMMENDED as it's now vulnerable
 to collision attacks [IACR-2019-459].
 
+# Use of Digest when acting on resources
+
+As the representation-data-digest is calculated on the selected representation,
+you need all the representation-metadata to compute it.
```

Comment:
Alternative suggestion:

A representation-data-digest is calculated on the selected representation, which requires complete representation metadata.